### DISCUSSION TITLE
leprzy-sposob-na-wyslanie-zdjec-przez-whatsapp/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#106] [General]
# leprzy-sposob-na-wyslanie-zdjec-przez-whatsapp/

Dzięki tej wskazówce poprawisz jakość udostępniania zdjęć w WhatsApp. Dowiedz się, jak wysyłać wysokiej jakości zdjęcia bez utraty danych oraz jak wysyłać wiele zdjęć jednocześnie.

https://dariusz.wieckiewicz.org/leprzy-sposob-na-wyslanie-zdjec-przez-whatsapp/

**Comment by xpil** [comment]
Najlepszy sposób na wysyłanie zdjęć przez WhatsApp to użycie Telegrama :) Pliki do 2TB (w wersji płatnej bodajże nawet do 4TB) i żadnych limitów na rozdzielczość.

**Comment by spexpl** [comment]
WhatsApp z oszczędzaniem danych i tak wychodzi chyba lepiej niż czysty mms.

**Reply by idarek** [reply]
> WhatsApp z oszczędzaniem danych i tak wychodzi chyba lepiej niż czysty mms.

A kto jeszcze używa MMS? 😉
